lib/block.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'room'
+module Hotels 
+	class Block_Of_Rooms < Room
```

Comment:
In general with Ruby we name classes using `CamelCase` rather than (capitalized) `Snake_Case`.